app/renderer/js/pages/preference/shortcuts-section.ts

```diff
@@ -30,35 +26,35 @@ export default class ShortcutsSection extends BaseSection {
 							<div class="settings-card">
 								<table>
 									<tr>
-										<td><kbd>${userOSKey}</kbd><kbd>,</kbd></td>
+										<td><kbd>${OSKey}</kbd> + <kbd>,</kbd></td>
 										<td>${t.__('Settings')}</td>
 									</tr>
 									<tr>
-										<td><kbd>${userOSKey}</kbd><kbd>K</kbd></td>
+										<td><kbd>${OSKey}</kbd> + <kbd>K</kbd></td>
 										<td>${t.__('Keyboard Shortcuts')}</td>
 									</tr>
 									<tr>
-										<td><kbd>${userOSKey}</kbd> + <kbd>Shift</kbd> + <kbd>M</kbd></td>
+										<td><kbd>${OSKey}</kbd> + <kbd>Shift</kbd> + <kbd>M</kbd></td>
 										<td>${t.__('Toggle Do Not Disturb')}</td>
 									</tr>
-									<tr>
-										<td><kbd>Shift</kbd><kbd>${userOSKey}</kbd><kbd>D</kbd></td>
+									<tr style= "display:${process.platform === 'darwin' ? '' : 'none'}">
```

Comment:
The empty string is not a valid value for the `display` property, and the `hidden` attribute is simpler:

```suggestion
									<tr ${process.platform === 'darwin' ? '' : 'hidden'}>
```